array-manipulation-1/src/compact.js

```diff
@@ -7,7 +7,9 @@
 function compact(array) {
   var hipHipArray = [];
   for (var i = 0; i < array.length; i++) {
-    if (!!array[i] === true) { hipHipArray.push(array[i]); }
+    if (!!array[i] === true) {
```

Comment:
You can also just do `if (array[i]) {` to check if the current value is truthy or not 🤓